.github/workflows/ci.yml

```
@@ -0,0 +1,20 @@
+---
+name: Validation
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+
+jobs:
+    validate:
+        runs-on: ubuntu-latest
+        steps:
+        - uses: actions/checkout@v2
```

```suggestion
        - uses: actions/checkout@v4
```
Let's use the latest :)